test/utils.js

```diff
@@ -147,7 +148,15 @@ const cleanRedisDb = async () => {
   })
 }
 
+const voteExchangeRate = 1 // 1 BAT per vote
```

Comment:
I would prefer we keep this in the e2e test. better to have the surveyor test just use it's own value so they're not interdependent

Reply:
that's fine. i can scope another one there. is there a good default that can be used for this fn @evq ?

Reply:
kinda depends what's under test.

e.g. in the surveyor test you could prob just use 1 vote and 1 probi - basically made up values since you aren't testing contributions and thus the exchange rate doesn't matter